test/test_helpers.h

```diff
@@ -105,6 +105,26 @@ inline void single_triangle_cw(const Vec3r<T> &center, Vec3rList<T> &vertices, V
   indices.emplace_back(Vec3ui{0, 1, 2});
 }
 
+template<typename T>
```

Comment:
Please leave comment regarding what the function does, even though it's rather simple.

I know, there is a lot of documentation missing in other places, but we are trying to improve that

Reply:
added some small comments for every function in test_helpers.h